node/utils.go

```diff
@@ -310,13 +311,22 @@ func createVolumeDir(hostPath, containerSpec, imageSpec, userSpec, permissionSpe
 	createVolumeDirMutex.Lock()
```

Comment:
i suppose we could create a lock per hostpath, but that seems somewhat risky this close to release.  With the other two optimizations that are less risky, we have already reduced all service container startup times from 10 minutes down to 1m30s.